package.json

```diff
@@ -25,13 +25,12 @@
   },
   "author": "Brainhub",
   "license": "MIT",
-  "dependencies": {},
   "devDependencies": {
     "autoprefixer": "^9.7.3",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
     "babel-eslint": "^10.0.3",
-    "babel-loader": "^8.0.6",
+    "babel-loader": "^7.1.5",
```

Comment:
why do you downgrade this?

Reply:
Downgraded babel-loader because v8 only supports Babel 7. 

I think migrating to Babel 7 would be better handled as a separate task.